README.rst

```diff
@@ -19,11 +19,12 @@
 Braindecode
 ===========
 
-`Braindecode`_ is an open-source Python toolbox for decode neurophysiological data such as MEG, EEG, sEEG,
-ECoG, and more with Deep Learning models. It includes modules for data input/output, preprocessing,
-visualization, data augmentation, classification, regression, sampler and much more!
+`Braindecode`_ is an open-source Python toolbox for decoding raw EEG data with
```

Comment:
I would keep ECoG as we have two examples analyzing ECoG data, for example this one: [plot_bcic_iv_4_ecog_cropped.py](https://github.com/braindecode/braindecode/blob/master/examples/plot_bcic_iv_4_ecog_cropped.py)

Reply:
Sure! Didn't know about those 😅  Cool stuff!